src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avernir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avernir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+html {
+  padding: 0;
+  box-sizing: border-box;
+  outline: none;
+}
+
+body {
+  margin-top: 20px;
+  font-family: Avernir, serif;
+}
+
+.big-input {
+  margin-bottom: 20px;
+  width: 100%;
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small-input {
+  margin-bottom: 20px;
+  width: 100%;
+  padding-left: 33px;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-input {
+  width: 100%;
+  box-sizing: border-box;
+  font-family: inherit, sans-serif;
+  font-weight: 300;
+  line-height: 21.86px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+}
+
+.search-input::placeholder {
+  font-weight: normal;
+}
+
+.big-form::before {
+  content: url(./images/Search.svg);
+  width: 19px;
+  height: 19px;
+  display: block;
+  position: absolute;
+  padding-left: 25px;
+  padding-top: 26px;
+}
+
+.small-form::before {
+  content: url(./images/Search.svg);
+  transform: scale(0.6);
+  width: 11px;
+  height: 11px;
+  display: block;
+  position: absolute;
+  padding-left: 13px;
+  padding-top: 15px;
+}
```

Comment:
Don't repeat yourself. You have a lot of duplicated properties. You can move it to common class and use this class